kolibri/plugins/user/assets/test/views/sign-up-page.spec.js

```diff
@@ -20,12 +20,6 @@ function makeWrapper() {
   return mount(SignUpPage, {
     store,
     router,
-    methods: {
```

Comment:
Again, tests were passing without problems when I removed this so I haven't replaced it though wanted to point it out.